src/components/Card.js

```diff
@@ -34,20 +44,25 @@ class Card {
 
     this._cardLikeElement.addEventListener("click", (evt) => {
       evt.stopPropagation();
+      this._toggleServerLike(this);
       this._toggleLike();
```

Comment:
`      this._toggleLike();
`

You need to **toggle likes** only in a `then` block after a successful response from the server because if there is an error, the user will not know about that and will see the toggled heart color.